### PR TITLE
[BugFix] Preserve backquote in Load Table command audit log

### LINKED ISSUE #22434: Preserve backquote in Load Table command audit log
When running a load table command like 
```
LOAD LABEL `development`.`table1`(
    DATA INFILE (
        's3://file_path'
    ) INTO TABLE table1 (
        `id`,
        `token`,
        `user_id`,
        `host`,
        `updated_at`,
        `deleted_at`
    )
    SET (ds = '2023-04-21')
) WITH BROKER broker_main PROPERTIES ("timezone" = "Etc/UCT", "timeout" = "36000")
```
In fe.audit.log the `stmt` will automatically remove the  ``` ` ```symbol in the fields, this may cause some issue when the table column name contains some reserved keyword, like `host`, because when we later parse the audit log stmt using `SqlParser.parse`, it will fail.

The ` symbol can be preserved in create table command, but not in load table command.

### Expected behavior (Required)
Preserve all ` symbol in audit log.

### Real behavior (Required)
The audit log currently is 
```
Stmt=LOAD LABEL `trust`.`table1`(DATA INFILE ('s3://file_path') INTO TABLE table1 (id, created_at, token, user_id, host, updated_at, deleted_at) SET (ds = '2023-04-23'))WITH BROKER broker_main PROPERTIES ("timezone" = "Etc/UCT", "timeout" = "36000")
```

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`
2.5.4

### PR DESCRIPTION
**Pull request by HangyuanLiu**
## What type of PR is this:
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes:

Fixes #22434 

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/sql/ast/DataDescription.java [hunk 1]
```diff
@@ -741,11 +744,15 @@ public String toString() {
         }
         if (fileFieldNames != null && !fileFieldNames.isEmpty()) {
             sb.append(" (");
-            Joiner.on(", ").appendTo(sb, fileFieldNames).append(")");
+            sb.append(Joiner.on(", ").join(
+                    fileFieldNames.stream().map(f -> "`" + f + "`").collect(Collectors.toList())));
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/sql/ast/DataDescription.java:748]: Add a backquote for every column name?

Comment (HangyuanLiu) [reply fe/fe-core/src/main/java/com/starrocks/sql/ast/DataDescription.java:748]: yes

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 16 / 19 (84.21%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/ast/ResourceDesc.java|0|1|00.00%|[85]|
|:large_blue_circle:|com/starrocks/qe/ConnectProcessor.java|0|1|00.00%|[212]|
|:large_blue_circle:|com/starrocks/sql/analyzer/AstToStringBuilder.java|6|7|85.71%|[386]|
|:large_blue_circle:|com/starrocks/sql/ast/DataDescription.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/analysis/BrokerDesc.java|1|1|100.00%|[]|

**Review by chaoyli — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**